src/Firebase.Database/JS.uno

```diff
@@ -74,8 +74,26 @@ namespace Firebase.Database.JS
         {
             var path = args[0].ToString();
             var key = args[1].ToString();
-            var val = args[2].ToString();
-            return new ReadByQueryEqualToValue(path,key,val);
+            var val = args[2];
+
+            if defined(iOS)
+            {
+                return new ReadEqualTo(
+                    path, key,
+                    JSON.ObjCObject.FromJSON(JSON.ScriptingValue.ToJSON(val))
+                );
+            }
+            else if defined(Android)
+            {
+                return new ReadEqualTo(
+                    path, key,
+                    JSON.JavaObject.FromJSON(JSON.ScriptingValue.ToJSON(val))
+                );
+            }
+            else
+            {
+                return new ReadEqualTo(path,key,val);
```

Comment:
Getting error  
`Error E3114: There is nothing named 'ReadEqualTo' accessible in this scope. Are you missing a package reference?`
I think you forgot to change function name in `Database.uno `

Reply:
my bad, fixed